### PR TITLE
First two wizard towers for Magiclysm

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
```SUMMARY: Mods "Adds wizard towers to magiclysm."```

#### Purpose of change
@KorGgenT asked me to make some wizard towers for magiclysm.

#### Describe the solution
This adds wizard tower 1, a normal house that happens to also be a stone wizard tower with a golem lab.

It adds plastic golems, a low-power but still formidable golem that can regenerate.

It also adds some furniture for magiclysm for wizard studies: an orrery, a standing alembic, magic work benches, and a huge mana crystal that glows.

#### Describe alternatives you've considered
I plan to add wizard tower 2, an office building style tower, and wizard tower 3, an improbably tall workshop.

EDIT: The office tower design isn't working so well, so I've changed it to a wizard tower that's also a house, a wizard tower that is mainly a workspace, and I'll probably build a third tower that belongs in the wilderness - but that might be a different PR.

#### Testing
I have been testing this in game as I go.

### CODE REVIEW

File: data/mods/Magiclysm/furniture.json [hunk 1]
```diff
@@ -50,5 +50,141 @@
...
+      "items": [
+        { "item": "pipe", "count": [ 6, 12 ] },
+        { "item": "cu_pipe", "count": [ 1, 4 ] },
+        { "item": "cable", "charges": [ 1, 4 ] },
+        { "item": "plastic_chunk", "count": [ 50, 75 ] },
+        { "item": "wood_panel", "count": [ 1, 2 ] },
+        { "item": "2x4", "count": [ 3, 4 ] }
+      ]
+    },
+    "max_volume": 7000,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/furniture.json]:
```suggestion
    "max_volume": "1750 L",
```

File: data/mods/Magiclysm/furniture.json [hunk 2]
```diff
@@ -50,5 +50,141 @@
...
+    "deconstruct": {
+      "items": [
+        { "item": "pot_copper", "count": 1 },
+        { "item": "pipe_glass", "count": [ 1, 4 ] },
+        { "item": "glass_tube_small", "charges": [ 5, 10 ] },
+        { "item": "stopcock", "count": [ 1, 5 ] },
+        { "item": "flask_glass", "count": [ 1, 3 ] }
+      ]
+    },
+    "max_volume": 7000,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/furniture.json]:
```suggestion
    "max_volume": "1750 L",
```

File: data/mods/Magiclysm/furniture.json [hunk 3]
```diff
@@ -50,5 +50,141 @@
...
+    "deconstruct": {
+      "items": [
+        { "item": "clockworks", "count": [ 10, 30 ] },
+        { "item": "scrap", "count": [ 10, 20 ] },
+        { "item": "pipe", "charges": 9 },
+        { "item": "wire", "count": [ 10, 20 ] },
+        { "item": "hinge", "count": [ 1, 3 ] }
+      ]
+    },
+    "max_volume": 7000,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/furniture.json]:
```suggestion
    "max_volume": "1750 L",
```

**Comment by KorGgenT** [comment]
image
image
image
image
image

**Comment by I-am-Erk** [comment]
Woops, I forgot to fix that on the git version. Done.

**Comment by KorGgenT** [comment]
image
these windows are all showers

**Comment by KorGgenT** [comment]
image
this one too, so uit must be a typo in the palette

**Comment by I-am-Erk** [comment]
Sure was. I added the shower as an afterthought and forgot to check if that letter was already used. As dumb a mistake as you might expect for having showers appear everywhere.

File: data/mods/Magiclysm/monsters/golems.json [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    "armor_cut": 6,
+    "melee_skill": 5,
+    "melee_dice": 3,
+    "melee_dice_sides": 6,
+    "melee_cut": 3,
+    "vision_day": 30,
+    "vision_night": 30,
+    "anger_triggers": [ "PLAYER_CLOSE", "HURT" ],
+    "regenerates": 10,
+    "death_drops": { "item": "plastic_chunk", "prob": 100, "count": [ 5, 10 ] },
```
Comment (ZhilkinSerg) [inline data/mods/Magiclysm/monsters/golems.json]: I believe `count` is not allowed here.

**Pull request merged**